aarch64-unknown-none.ld

```diff
@@ -6,7 +6,7 @@ ENTRY(ram64_start)
     ACPI:   [0x4020_0000-0x403f_ffff)
     kernel: [0x4048_0000-]
    The stack start is at the end of the DRAM region. */
```

Comment:
```suggestion
    payload:[0x4040_0000-0x405f_ffff)
    RHF:    [0x40600000-]
   Assuming 2MB is enough to load payload.
   The stack start is at the end of the DRAM region. */
```
Add comments on the memory layout assumption.

Reply:
OK.